provider/provider.go

```diff
@@ -102,12 +102,25 @@ const (
 type InitConfig struct {
 	// This is the location of the code base that the
 	// Provider will be responisble for parsing
+	// Deprecated: rootUri, which is what this maps to in the LSP spec, is deprecated.
```

Comment:
Can you create issues in the repo for this and link in the deprecated or a TODO?

Reply:
Done! https://github.com/konveyor/analyzer-lsp/issues/365